Composer/packages/ui-plugins/schema-editor/src/AdaptiveDialogField.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React from 'react';
+import { JsonEditor } from '@bfc/code-editor';
+import { FieldLabel, schemaField, ObjectField } from '@bfc/adaptive-form';
+import { FieldProps, useShellApi } from '@bfc/extension';
+import formatMessage from 'format-message';
+
+const getDefaultDialogSchema = (title: string, description = '') => ({
```

Comment:
Not sure if there is a better place for the `getDefaultDialogSchema` function to live.

Reply:
This is fine.